libs/forms/src/validators/not-empty.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  AbstractControl,
+  FormGroup,
+  ValidationErrors,
+  ValidatorFn,
+} from '@angular/forms';
+
+/**
+ * A validator for a form group that returns an error if all of its immediate children don't have a truthy value.
```

Comment:
This doesn't return an error.

What do you think about:

"A form validator that passes validation when the Form control or at least one of the immediate children of a Form group have a value."

Reply:
changed